dev-utils/poly-cli/src/static/templates/preview/styles.css

```diff
@@ -41,8 +41,7 @@
 }
 
 .preview h1 {
-    font-family: var(--jost);
-    font-weight: var(--poly-font-weight-medium);
+    font-family: var(--jost-medium);
```

Comment:
You can remove this `font-family`, it's already specified in polyLook 
at `.poly-standard-layout h1`